en/2-build-in-accessibility-from-start.md

```diff
@@ -116,14 +149,15 @@ Your service must be accessible to users regardless of their digital confidence
 
 **[TODO: Add/revise checklist items]**
 
-- meet the [Standard on Web Accessibility](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=23601)
+- {: .dpgn-accessibility } meet the [Standard on Web Accessibility](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=23601)

```

Comment:
This tag is different than the others. Is that intentional? It is ".dpgn-accessibility" while the others seem to be ".dpgn-accessibility-usability',

Reply:
Confirmed that is intended to be different.